go.mod

```diff
@@ -4,9 +4,9 @@ go 1.19
 
 require (
 	github.com/aws/aws-sdk-go v1.55.5
-	github.com/gofrs/uuid v4.4.0+incompatible
+	github.com/gofrs/uuid/v5 v5.3.0
```

Comment:
How is this different from `github.com/gofrs/uuid v5.3.0`?

Reply:
`v4.4.0+incompatible` is != `v5.3.0`. The commits are different.
`v5.3.0` is the latest version with no blackduck scan issues reported. Hope this answers your Q.

Reply:
> `v4.4.0+incompatible` is != `v5.3.0`. The commits are different.

I know, but..

> `v5.3.0` is the latest version with no blackduck scan issues reported. Hope this answers your Q.

I was wondering about the extra `v5` in `github.com/gofrs/uuid/v5 v5.3.0`. Is there something wrong with `github.com/gofrs/uuid v5.3.0`?

Reply:
Ahh I see. `github.com/gofrs/uuid/v5` only works. I tried `go get github.com/gofrs/uuid@v5.3.0`, but it threw an error message telling me to use `github.com/gofrs/uuid/v5`.

Reply:
@anoopcs9 take a look at https://www.alexedwards.net/blog/an-introduction-to-packages-imports-and-modules (jump to  "Version 2+ packages")

Reply:
> @anoopcs9 take a look at https://www.alexedwards.net/blog/an-introduction-to-packages-imports-and-modules (jump to "Version 2+ packages")

Oh..it is more like library versioning, I get it now. Thanks.